jsonforms-tooling-common/src/index.ts

```diff
@@ -9,85 +9,110 @@ import { sep } from 'path';
 const npm = process.platform === 'win32' ? 'npm.cmd' : 'npm';
 
 /*
- * Clones a git repository and runs npm install on it
- * @param {string} repo the name of the repo that should be cloned
- * @param {string} path to the folder, where the repo should be cloned into
- * @param {function} callback forwards the current status to the caller
+ * Receives the data from the editor and calls the install methos
+ * @param {any} editorInstance the instance of the editor
+ * @param {any} args the arguments passed to the editor call
+ * @param {string} project the project, that should be installed
  */
-export const cloneAndInstall = (
-  repo: string,
-  path: string,
-  callback: (id: string, result: string, type?: string) => void,
-  name?: string) => {
-  let url = '';
-  switch (repo) {
-    case 'example':
-      url = 'https://github.com/eclipsesource/make-it-happen-react';
-      break;
-    case 'seed':
-      url = 'https://github.com/eclipsesource/jsonforms-react-seed';
-      break;
-    default:
-      return;
+export const createProject = (editorInstance: any, args: any, project: string) => {
```

Comment:
`args` is actually only needed to determine the file path (`fsPath`), right? And `project` is either `"seed"` or `"example"` for which I'd prefer to use an enum. Please update the signature accordingly.